docs/advanced/storagenetwork.md

```diff
@@ -290,6 +290,101 @@ metadata:
 Omitted...
 ```
 
+#### Step 4
+
+The Storage Network relys on the external network to work. When the external network is not ready, you may encounter those errors:
```

Comment:
Can you describe the use of "external network" in more detail? A storage network is typically an internal and dedicated network to the cluster nodes for storage traffic.

Reply:
@bk201 Please refer the two pictures on https://docs.harvesterhci.io/v1.3/advanced/storagenetwork#same-physical-interfaces, the storage network are physically connected by those external switchs/routers, user needs to configure them correctly. One customer had ever reported such issue and finally toubleshooted that their switchs were not configured well.

Reply:
I feel we could elaborate more on the paragraph. Generally, the storage network is a particular case of VM Networks (NADs). They all involve uplink configuration and rely on the network infrastructure to provide connectivity. The only requirement is that the Cluster Network the storage network is associated with must have full coverage across the entire cluster. That connectivity is also provided by the network infrastructure because Harvester nodes are not directly connected to each other. To make it easier to grasp, we could focus on the main idea of L2 connectivity that must be ensured, which implies the user has to configure the network infrastructure accordingly.

With that said, it's okay to leave the term "external network". But I think the reference link should be https://docs.harvesterhci.io/v1.3/networking/deep-dive#external-networking

Reply:
I updated the statement & link on Step 4, please take another look, thanks.